### PR TITLE
Removed the part of the subnets creation

### PR DESCRIPTION
**Pull request by s2504s**
## What
* Removed the part of the subnets creation

## Why
* We should be able to create a `vpc` only

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,21 +1,30 @@
 variable "namespace" {
-  default = "global"
+  default = ""
```
Comment (osterman) [inline variables.tf]: Remove this default

File: variables.tf [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
 }
 
 variable "stage" {
-  default = "default"
+  default = ""
```
Comment (osterman) [inline variables.tf]: Remove this default

File: variables.tf [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 }
 
 variable "name" {
-  default = "vpc"
-}
-
-variable "availability_zones" {
-  description = "Run the EC2 Instances in these Availability Zones"
-  type        = "list"
+  default = ""
```
Comment (osterman) [inline variables.tf]: Remove this default

**Comment by osterman** [comment]
missing `depends_on` for IGW

**Comment by osterman** [comment]
This PR needs a rebase. Also, the description is wrong. We utilize `tf_subnets` in https://github.com/cloudposse/tf_vpc/pull/4

**Comment by comeanother** [comment]
Invalidated by https://github.com/cloudposse/tf_subnets/pull/15

**Pull request closed without merging**